google-cloud-bigtable-stats/pom.xml

```diff
@@ -230,6 +237,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <version>2.0.0</version>
+                <executions>
+                    <execution>
+                        <id>default-cli</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>add-third-party</goal>
+                        </goals>
+                        <configuration>
+                            <excludedScopes>test</excludedScopes>
+                            <includedGroups>io.opencensus:*</includedGroups>
```

Comment:
please add a note that this should be kept in sync with the shading config & add a reverse note there